debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java

```diff
@@ -341,22 +341,22 @@ else if (updatedRows != 1){
 
         // update overall stats
         LocalDateTime now = LocalDateTime.now(ZoneOffset.UTC);
-        LocalDateTime nowFlooredToNearest30s = now.minusSeconds(now.getSecond()%30);
-        Long nowFlooredToNearest30sEpoch = nowFlooredToNearest30s.toEpochSecond(ZoneOffset.UTC);
+        LocalDateTime nowFlooredToNearest10s = now.minusSeconds(now.getSecond()%10);
```

Comment:
No need to change it now: but the ideal code structure should have the STATS_PUBLISHING_INTERVAL as a constant and one should only need to change its value to change the publishing interval. That also means the variable names should not be tied to the interval e.g. `currentIntervalStart`

Reply:
Right, I guess my intention was to make the interval explicit in the var name, but what you suggested makes more sense!